hw/ip/otbn/doc/fcov.md

```diff
@@ -0,0 +1,239 @@
+---
+title: OTBN functional coverage plan
```

Comment:
The DV doc is precisely the place to capture this. I agree with @matutem - there is no need to put this in a separate document.

Reply:
It's going to get awfully long: I suspect we'll change our mind about this at some point. But ok.